package com.emtech.dairyapp.Stock.Data.Http.Request.Category;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.io.Serializable;

@ToString
@Data
@AllArgsConstructor
@NoArgsConstructor
public class CategoryCreateRequest implements Serializable {
    @JsonProperty(value = "name")
    private String name;

    @JsonProperty(value = "description")
    private String description;
}
